### ISSUE TITLE
Saving images fails randomly

### ISSUE THREAD

**Issue by SirusCodes** [SirusCodes/Heartry#5]
While saving/sharing images android platform fails and fails to save the image.

**Possible soultions**
1. Find a better plugin to save images
2. Create one by ourselves

**Comment by uveshm003** [comment]
What issue are you facing can you please specify so I can focus on that particular test case only.
I am having a very good knowledge of working with files in flutter.

**Comment by SirusCodes** [comment]
> What issue are you facing can you please specify so I can focus on that particular test case only.
> I am having a very good knowledge of working with files in flutter.

Honestly I'm not able to figure it. But can you implement saving photos into the gallery?

**Comment by uveshm003** [comment]
Yeah sure and also the on the  other Two issue Font related and search related I mistakenly commented using my work account so can you please remove that account and assign those issues to this account.

**Comment by uveshm003** [comment]
Also can you please write flutter version that is compatible with these project in Contributing.md file.

**Comment by uveshm003** [comment]
Okay then assign this to me as well I will do it this week. and also on the other Two issue Font related and search related I mistakenly commented using my work account so can you please remove that account and assign those issues to this account.
Account name is uveshmenpur can you remove that and assign those issues to this account

**Comment by SirusCodes** [comment]
> and also on the other Two issue Font related and search related I mistakenly commented using my work account so can you please remove that account and assign those issues to this account.

Can you comment with this account on those so I can assign it?

**Comment by uveshm003** [comment]
sure
